plugin/build.gradle.kts

```diff
@@ -156,7 +156,7 @@ tasks.withType<Test>().configureEach {
     useJUnitPlatform()
 }
 
-tasks.test {
+tasks.withType<Test> {
```

Comment:
This ensures that the system properties are available for all test tasks, not just the default one